### ISSUE TITLE
On the Contributors page, the Tracks are not sorting properly

### ISSUE THREAD

**Issue by glennj** [exercism/exercism#5922]
https://exercism.org/contributing/contributors?

<img width="261" alt="contributors 1" src="https://user-images.githubusercontent.com/122470/133653107-b87a5a4f-486c-4555-bb6f-3eca4819eb10.png">
<img width="261" alt="contributors 2" src="https://user-images.githubusercontent.com/122470/133653114-ecb5b880-8004-4d5d-8973-5057bcbd8966.png">

**Comment by iHiD** [comment]
Some nice renames showing their colours there (e.g. JavaScript used to be EMCAScript!). Thanks - will sort.

**Comment by glennj** [comment]
3 of "my" tracks at the bottom there, so I was feeling a little excluded (boohoo)

**Comment by iHiD** [comment]
Yeah, it's the order they were added to the database by the look of it.
